src/db/connection.js

```diff
@@ -12,11 +15,11 @@ const connectToMongo = () => {
   const db = mongoose.connection;
 
   db.once('open', () => {
-    console.log('Database connected to: ', url);
+    logger.log('info', 'Database connected to: ', url);
```

Comment:
Can you use the backticks syntax here similar to the log statement in `app.js`?
Because winston logger doesn't seem to accept this syntax as the URL is not showing in the log statements.